### ISSUE TITLE
Toaster uses vanilla wooden button and note block SoundEvents, should use its own for resource pack compatibility

### ISSUE THREAD

**Issue by saltyseadoggo** [FoundationGames/Sandwichable#53]
When I was messing with replacing the button press sounds with a "Boop!" sound effect for comedic effect, I noticed that the Toaster had the custom sounds applied to it as well, with a problematically warped pitch. I looked through the mod's _sounds.json_ and noted that there are no entries for the Toaster's sound events, and then checked _ToasterBlockEntity.java_ in the source code to confirm that the Toaster uses the vanilla SoundEvents for pressing a wooden button and playing a note block set to the bell instrument.

I would like to request that the Toaster use its own SoundEvents with their own entries in the mod's _sounds.json_. The sounds can by all means stay the same; I would simply like to see these sound events separated from the vanilla button and note block ones so that resource pack creators can re-map them independantly.~

**Comment by FoundationGames** [comment]
Yes, I didn't want to add unnecessary sounds when the vanilla ones work fine. To add to this, the Sandwich Maker villager's work sound is the item frame remove item sound effect, and the cutting board sound is a pitched up pumpkin carve sound. I might create custom sound events that use the same audio clips, just so resource pack makers can remap them.

**Comment by saltyseadoggo** [comment]
That's what I was suggesting: just adding custom SoundEvents that use the same sounds these features already use.~
